app/parse.py

```diff
@@ -15,8 +36,97 @@ class Product:
     num_of_reviews: int
 
 
+FIELD_NAMES = [field.name for field in fields(Product)]
+
+
+def initialize_driver() -> webdriver.Chrome:
+    chrome_options = webdriver.ChromeOptions()
+    chrome_options.add_argument("--headless")
+    driver = webdriver.Chrome(options=chrome_options)
+    return driver
+
+
+def cookies_button(driver: webdriver.Chrome) -> None:
```

Comment:
```suggestion
def cookies_button(driver: WebDriver) -> None:
```
It's recommended to use a more generic type like WebDriver instead of Chrome. Fix throughout the code.